modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx

```diff
@@ -369,9 +369,9 @@ export default function EditObjectField({
 								value:
 									objectField.businessType === 'Integer' ||
 									objectField.businessType === 'LongInteger'
-										? // @ts-ignore
-
-										  parsedFilter.json[filterType]
+										? (parsedFilter.json as {
+												[key: string]: string;
+										  })[filterType]
```

Comment:
Hey @naraandrade, I only removed this one, because the others will be removed here: https://github.com/liferay-objects/liferay-portal/pull/867